src/Data/Vec/Relation/Binary/Lex/Strict.agda

```diff
@@ -139,7 +140,7 @@ module _ {_≈_ : Rel A ℓ₁} {_≺_ : Rel A ℓ₂} where
     where
 
     <-wellFounded : ∀ {n} → WellFounded (_<_ {n})
-    <-wellFounded {0}     [] = acc λ ys<[] → ⊥-elim (xs≮[] ys<[])
+    <-wellFounded {0}     [] = acc λ ys<[] → contradiction ys<[] xs≮[]
```

Comment:
and also remove import of `⊥-elim`?

Reply:
Oops... no can do L209-L210: 
```agda
  <⇒≤ : ∀ {m n} {xs : Vec A m} {ys : Vec A n} → xs < ys → xs ≤ ys
  <⇒≤ = Core.map-P ⊥-elim
```
Suggest merging as is!